brianlong/www.validators.app
# 183114541 endpoint for a map

#### What's this PR do?
- Create endpoint providing data for a map

#### How should this be manually tested?
- run migrations
- run tests
- run DataCenterStatsWorker to populate data_center_stats
- check endpoint `/api/v1/data-centers-with-nodes/mainnet`

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183114541)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
